src/main/java/org/kairosdb/core/tiers/AccessTokenProvider.java

```diff
@@ -0,0 +1,26 @@
+package org.kairosdb.core.tiers;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.google.inject.name.Named;
+import org.zalando.stups.tokens.AccessTokens;
+import org.zalando.stups.tokens.Tokens;
+
+import java.net.URI;
+
+public class AccessTokenProvider implements Provider<AccessTokens> {
+
+    private final String tokensUri;
+
+    @Inject
+    public AccessTokenProvider(@Named("tokens.uri") String tokensUri) {
+        this.tokensUri = tokensUri;
+    }
+
+    @Override
+    public AccessTokens get() {
+        return Tokens.createAccessTokensWithUri(URI.create(tokensUri))
+                .manageToken("zmon-read")
```

Comment:
should be configurable

Reply:
why? This (to my understanding) corresponds to the entry of `PlatformCredentialsSet` which is constant for application,